docs/license.md

```diff
@@ -1,23 +1,30 @@
 # License
 
-we have a universal donor policy, our code should be able to be used anywhere by anyone, no ifs or conditions attached.
+We have a universal donor policy, our code should be able to be used anywhere by anyone, no ifs or conditions attached.
 
-We go to great lengths to avoid GPL code for this reason, all our code is either Apache2 or BSD licensed
+OVOS is predominately Apache2 or BSD licensed. There are only a few exceptions to this, which are all licensed under other compatible open source licenses.
 
-Individual plugins or skills may have their own license, for example mimic3 is AGPL so we can not change the license of
-our plugin.
+Individual plugins or skills may have their own license, for example mimic3 is AGPL so we can not change the license of our plugin.
 
-We are committed to maintain all core components fully free, any GPL code will live in an optional plugin and be flagged
-as such.
+We are committed to maintain all core components fully free, any code that we have no control over the license will live in an optional plugin and be flagged as such.
+
+This includes avoiding LGPL code for reasons explained [here](https://softwareengineering.stackexchange.com/questions/119436/what-does-gpl-with-classpath-exception-mean-in-practice/326325#326325).
+
+Our license policy has the following properties:
+
+-  It gives you, the user of the software, complete and unrestrained access to the software, such that you may inspect, modify, and redistribute your changes
+  - Inspection - Anyone may inspect the software for security vulnerabilities
+  - Modification - Anyone may modify the software to fix security vulnerabilities
+  - Redistribution - Anyone may redistribute the software to patch the security vulnerabilities
```

Comment:
`Anyone may redistribute the software on their terms`? They can redistribute under whatever license they want or not